examples/tests/responses/test_background_tasks.py

```diff
@@ -0,0 +1,40 @@
+from unittest.mock import patch
+
+from examples.responses.background_tasks_1 import app as app_1
+from examples.responses.background_tasks_2 import app as app_2
+from examples.responses.background_tasks_3 import app as app_3
+from examples.responses.background_tasks_3 import greeted as greeted_3
+from starlite import TestClient
+
+
+def test_background_tasks_1() -> None:
+    with TestClient(app=app_1) as client, patch("examples.responses.background_tasks_1.logger.info") as mock_info:
```

Comment:
Use the [`caplog`](https://docs.pytest.org/en/7.1.x/how-to/logging.html#caplog-fixture) fixture here instead of mocking. Applies to the rest of the tests as well.

Reply:
Done in latest commit. Thank you for this suggestion, I did not know about `caplog` 👍